src/modules/dashboard/components/Extensions/ExtensionSetup.tsx

```diff
@@ -201,6 +201,17 @@ const ExtensionSetup = ({
     whitelistAddress,
   ]);
 
+  const showInputField = useCallback(
```

Comment:
It works, but this is kinda super specific for Whitelist.

I which there was a better way to extension specific cases like this somewhere else, but I can't really think of an alternative :(

Reply:
I know, I don't like it too much myself but I thought about other options and couldn't come up with a better solution...